hw3/01-characters.js

```diff
@@ -2,3 +2,55 @@
 const url = 'https://thronesapi.com/api/v2/Characters';
 
 fetch(url);
+
+const characterFlex = document.getElementById("charcter-items");
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    data.forEach((c) => {
+        let characterCard = document.createElement("div");
+        characterCard.id = "thrones-characters";
+        characterCard.classList = "col p-2 bg-transparent border-0 mx-auto";
+
+        let characterImageDiv = document.createElement("div");
+        characterImageDiv.classList = "row h-75";
+
+        let characterImage = document.createElement('img');
+        characterImage.id = "thrones-img";
+        characterImage.classList = "h-100";
+        characterImage.alt = "image";
+        characterImage.src = c['imageUrl'];
+
+        characterImageDiv.appendChild(characterImage);
+        characterCard.appendChild(characterImageDiv);
+
+        let characterDetails = document.createElement("div");
+        characterDetails.classList = "row h-25 text-center p-2 mx-auto";
+
+        let characterNameTitleDiv = document.createElement("div");
+        characterNameTitleDiv.classList = "col";
+
+        let characterName = document.createElement('p');
+        characterName.classList = "fw-bold mx-auto mb-0";
+        characterName.style = "font-size: 0.9rem;";
+        characterName.textContent(c["fullName"]);
```

Comment:
![image](https://github.com/Rajashree-Pethkar/frontend-homework/assets/38127011/4131541b-a20d-43e0-b986-c8c42563f7a1)

I got this error in the console.

Is everything working correctly on your end?

This implementation only works on my end if I change it to something like 

`characterName.textContent = c["fullName"];`